devito/core/cpu.py

```diff
@@ -254,15 +254,24 @@ class Intel64FSGOpenMPOperator(Intel64FSGOperator, CPU64OpenMPOperator):
 class CustomOperator(CPU64Operator):
 
     _known_passes = ('blocking', 'denormals', 'optcomms', 'openmp', 'mpi',
-                     'simd', 'prodders', 'topofuse', 'fuse')
+                     'simd', 'prodders', 'topofuse', 'fuse', 'factorize',
```

Comment:
all these changes are cherry-pick leftovers. They won't harm -- I'll need them for the performance notebook PR